VikingQuiz.UI/src/Components/QuizItem/QuizItem.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import './QuizItem.css';
+import ManageQuizComponent from '../MyQuizesPage/ManageQuizComponent/ManageQuizComponent';
+
+
+class QuizItem extends React.Component<any, any>{
+    constructor(props: any) {
+        super(props);
+  
+        this.state = {
+            serverMessage: '',
+            redirect: false,
+            isHidden: true,
+            id: this.props.id,
+            handleChildDelete: this.props.handleChildDelete,
+            isSelected: false
+        }
+
+        this.handleMouseHover = this.handleMouseHover.bind(this);
+        this.handleMouseOut = this.handleMouseOut.bind(this);
+        this.handleDeleteSelection = this.handleDeleteSelection.bind(this);
+        this.componentWillMount = this.componentWillMount.bind(this);
+      }
+
+      public componentWillMount(){
+          this.setState({isSelected:false})
+      }
+
+      public handleMouseHover(){
+            this.setState({
+                isHidden: false
```

Comment:
You can write this in a single line. If you're feeling really adventurous you can even write it as:
`public handleMouseHover() { this.setState({isHidden: false}); }`